Mathlib/Data/List/Chain.lean

```diff
@@ -164,7 +164,7 @@ theorem chain_iff_get {R} : ∀ {a : α} {l : List α}, Chain R a l ↔
 set_option linter.deprecated false in
 @[deprecated chain_iff_get]
 theorem chain_iff_nthLe {R} {a : α} {l : List α} : Chain R a l ↔
-        (∀ h : 0 < length l, R a (nthLe l 0 h)) ∧
+    (∀ h : 0 < length l, R a (nthLe l 0 h)) ∧
           ∀ (i) (h : i < length l - 1),
             R (nthLe l i (lt_of_lt_pred h)) (nthLe l (i + 1) (lt_pred_iff.mp h)) :=
```

Comment:
Could you fix this part of the statement too?

Reply:
done